pkg/api/getambassador.io/v3alpha1/crd_authservice.go

```diff
@@ -30,6 +30,8 @@ type AuthServiceIncludeBody struct {
 
 	// +kubebuilder:validation:Required
 	AllowPartial bool `json:"allow_partial,omitempty"`
+
+	BodyAsRawBytes *bool `json:"body_raw_bytes,omitempty"`
```

Comment:
1. Because the default value (false) and the zero value are the same, no need to make it a pointer.
2. This is a little hypocritical because none of the other fields have comments right now, but can you please add a doc-comment?
3. The name seems funny. Because this is already inside of the `include_body` field, including `body_` in the name is redundant. `body_raw_bytes` sounds to me like it is of type `[]byte`, not `bool`. I'm thinking `as_bytes` is a better name?

So something like

```go
// AsBytes (default=false) controls whether to transmit the body as bytes (true) or as a
// string (false).  Request data that is not valid UTF-8 cannot be transmitted as a string,
// and will result in a 403 UAEX response if AsBytes is not set to true.
AsBytes bool `json:"as_bytes,omitempty"`

```